test/unit/document_test.rb

```diff
@@ -152,4 +152,17 @@ class DocumentTest < ActiveSupport::TestCase
 
     refute draft.document.similar_slug_exists?
   end
+
+  test "current government can only be set once" do
```

Comment:
This test description suggests to me there's some sort of validation or check on the government not being updated, which isn't the case.

How about something like:

```ruby
test "#ensure_document_has_government does not change existing government association" do
  …
end
```